automatic/utorrent/tools/ChocolateyInstall.ps1

```diff
@@ -0,0 +1,16 @@
+﻿$packageArgs = @{
+    packageName    = $env:ChocolateyPackageName
+    softwareName   = 'uTorrent*'
+    url            = 'http://download.ap.bittorrent.com/track/stable/endpoint/utorrent/os/windows'
+    checksum       = '154442d57d5f785c73a20efc3f83ce95c87de46ad057eb8f386c6a451028c34e'
+    fileType       = 'exe'
+    checksumType   = 'sha256'
+    silentArgs     = '/S'
```

Comment:
If utorrent by default installs to the appdir directory, then the silent args should be updated with an override to install into a sane directory instead.
for instance update to:
`silentArgs     = "/S /D=$(Get-ToolsLocation)\uTorrent"`

That just an example though